readme.md

```diff
@@ -33,7 +33,7 @@ consume this package.
 ## Install
 
 This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
-In Node.js (version 12.20+, 14.14+, or 16.0+), install with [npm][]:
+In Node.js (version 14.14+, or 16.0+), install with [npm][]:
```

Comment:
```suggestion
In Node.js (version 14.14+, 16.0+, or 18.0+), install with [npm][]:
```

Reply:
Oops, this should have been part of #8.

Anyway, do you really want to explicitly state Node.js 18.0+ is supported if it’s already specified Node.js 16.0+ is supported? I get the point for Node 12 and 14, as they explicitly required minimal minor versions, but now IMO it’s getting redundant to mention every major Node.js version if we don’t rely in such specific features introduced in minor versions. I’m pretty sure most packages will work with future versions of Node.js.

Reply:
Particularly in the compat docs, mentioning them a) helps clarify that this is modern and tested, b) the consistency with every project listing the supported release lines is nice, c) it makes future edits easier/smaller/clearer, e.g., when it’s `18.5+` or so

It’s less important in the install section, maybe just `(version 14.14 or later)` there is okay. On the other hand, it’s fine duplicating some info: users might see install but not look at compat